lib/linguist/vendor.yml

```diff
@@ -349,23 +349,23 @@
 - ^Vagrantfile$
 
 # .DS_Stores
-- .[Dd][Ss]_[Ss]tore$
+- \.[Dd][Ss]_[Ss]tore$
 
 # R packages
 - ^vignettes/
 - ^inst/extdata/
 
 # Octicons
-- octicons.css
-- sprockets-octicons.scss
+- (^|/)octicons\.css
```

Comment:
If they're all interpreted as regular expressions, shouldn't we add this `(^|/)` pattern in front of all entries (e.g., `stuff\.framework` entries above)?

Reply:
That's a huge maybe from me 😄 

I'm not 100% sure of all the strings that can be used, so I didn't want to do this to them all in one go.

Reply:
Ok. How did you choose where to add `(^|/)` and where not to add it then?

Reply:
They were the ones I either knew about or was able to find out about quickly. I didn't do an exhaustive search to check the others.

Reply:
I think it's okay to add it everywhere unless it's obvious that the expression should match the end of a filename (e.g., file extension). I'm not sure we have any such case anyway. Worst-case scenario, we break one or two of those and we'll fix it quickly afterward when users come complain.